app/controllers/users_controller.rb

```diff
@@ -1,4 +1,26 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
   end
+
+  def show
+    @user = User.find_by params[:id]
```

Comment:
find_by id: params[:id], what is different between find_by and find

Reply:
find will return an exception if no record found => server maybe crash, and find_by return nil